public/static/admin/config.yml

```diff
@@ -199,6 +199,8 @@ collections:
     label_singular: "Blog Post"
     folder: "content/blog/posts"
     create: true
+    slug: "{{year}}-{{month}}-{{day}}_{{title}}_{{authors}}"
```

Comment:
Is this what we'll use for blog post URLs? I wonder if we need all the date information in there. It looks like Netlify CMS has some sensible defaults for slugs, like [using the title field](https://www.netlifycms.org/docs/configuration-options/#identifier_field).